lib/react-popout.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
```

Comment:
I think this should be renamed back to .jsx, it's more obvious and additionally if anyone was importing this type directly this will be a breaking change